src/api/spec/controllers/staging/excluded_requests_controller_spec.rb

```diff
@@ -18,9 +20,22 @@
            review_by_group: group)
   end
 
-  before { login(manager) }
+  describe 'GET #index' do
+    before do
+      get :index, params: { staging_main_project_name: staging_workflow.project.name, format: :xml }
+    end
+
+    it { expect(response).to have_http_status(:success) }
+    it 'returns the excluded_requests xml' do
+      assert_select 'excluded_requests' do
+        assert_select 'request', 0
```

Comment:
I didn't see it in time, but this test is not enough. How are we sure it's working with this?

Reply:
that was exactly my previous comment, I don't understand why this got merged without that issue being discussed.

Reply:
I will create another pull request to fix the test.

Reply:
thanks @eduardoj